src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todo Elmar</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  
+  <div class="flex-wrapper">
+    <h1>Today's To Do</h1>
+    <div class="add-todo-wrapper">
+      <div class="input-wrapper">
+        <input type="text" id="description-task" placeholder="Add to your list">
+      </div>
+      <div class="button-wrapper">
+        <button id="add-task-btn" type="button">Add</button>
+      </div>
+    </div>
+    
+    <ul class="todos-wrapper">
+      
+    </ul>
```

Comment:
You may consider closing up an empty element. Since the element does not have any content, I think it is cleaner to leave it closed.
Eg.
```
<ul class="todos-wrapper"></ul>
```
what do you think?

Reply:
Done